src/main/java/org/apache/cassandra/service/StorageService.java

```diff
@@ -1419,10 +1419,13 @@ public void rescheduleFailedDeletions() {
      *            The ColumnFamily name where SSTables belong
      */
     @Override
-    public void loadNewSSTables(String ksName, String cfName) {
+    public void loadNewSSTables(String ksName, String cfName, boolean isLoadAndStream) {
         log(" loadNewSSTables(String ksName, String cfName)");
         MultivaluedMap<String, String> queryParams = new MultivaluedHashMap<String, String>();
         queryParams.add("cf", cfName);
+        if (isLoadAndStream)
```

Comment:
Java convention is to always use a {} block after `if`

Reply:
Not in this repo :)
but fixed